### PR TITLE
 Update link 4devs in Other Free Resources  and Delete deploy now service down

### PR DESCRIPTION
**Pull request by CristianoFIlho**
## Requirements

 * [X] Deletion of deploy now service down
 * [X] It has a free tier not just a free trial
 * [X] Free online tools, cpf generators, bank account, people, cnpj, zip code, rg, validators, encoders and many other useful tools for programmers.
 * [X] The submission is not already present in the list
 * [X] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -30,8 +30,8 @@
 <!-- This is only for new submissions -->
 <!-- Please ensure your submission ticks all of the requirements -->
 
- * [ ] This is Software as a Service not self hosted
- * [ ] It has a free tier not just a free trial
- * [ ] The submission mentions what is free
- * [ ] The submission is not already present in the list
- * [ ] The service has contact details of those running it and a privacy policy
+ * [X] Deletion of deploy now service down
```
Comment (ripienaar) [inline .github/PULL_REQUEST_TEMPLATE.md:33]: Please do not edit this file

File: README.md [hunk 2]
```diff
@@ -1731,6 +1730,7 @@ HTML Themes and UI Kits to create your applications faster!
   * [Utils.fun](https://utils.fun/en) — All offline daily and development tools based on the browser's computing power, including watermark generation, screen recording, encoding and decoding, encryption and decryption and code formatting, are completely free and do not upload any data to the cloud for processing.
   * [QR Code Robot](https://qr-code-robot.com) — Free QR Code Generator. Offers possibilty to generate a QR Code for a webpage, sms, email, wifi auth or free text. Plans to offer also a generator for file sharing (pdf, mp3, video) and also a scan tracking of QR code.
 * [Free Code Tools](https://freecodetools.org/) — Effective code tools which are 100% free. Markdown editor, Code minifier/beautifier, QR code generator, Open Graph Generator, Twitter card Generator and more.
+* [4devs](https://www.4devs.com.br/) — Effective code tools for Brazilian developers 100% free, validations and generator tools.
```
Comment (ripienaar) [inline README.md:1733]: We can only accept english language services, do they have a english version?

File: README.md [hunk 3]
```diff
@@ -1048,7 +1048,6 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [Alwaysdata](https://www.alwaysdata.com/) — 100 MB free web hosting with support for MySQL, PostgreSQL, CouchDB, MongoDB, PHP, Python, Ruby, Node.js, Elixir, Java, Deno, custom web servers, access via FTP, WebDAV and SSH; mailbox, mailing list and app installer included.
   * [Awardspace.com](https://www.awardspace.com) — Free web hosting + a free short domain, PHP, MySQL, App Installer, Email Sending & No Ads.
   * [Bubble](https://bubble.io/) — Visual programming to build web and mobile apps without code, free with Bubble branding.
-  * [Deploy Now](https://deploynow.space) — Deploy smarter. Deploy faster. Deploy Now. - Deploy up to 3 web projects from your GitHub repository for free.
```
[the remark this answers is not in the document]
Comment (ripienaar) [reply README.md:1051]: but no longer free, so we can remove it

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (CristianoFIlho) [reply .github/PULL_REQUEST_TEMPLATE.md:33]: only in portuguese

File: README.md (hunk 2 above)
Comment (CristianoFIlho) [reply README.md:1733]: only in portuguese

**Pull request closed without merging**